specs/electra/validator.md

```diff
@@ -194,7 +194,7 @@ in [EIP-7685](https://eips.ethereum.org/EIPS/eip-7685). The index of each elemen
 2. Set `block.body.execution_requests = get_execution_requests(execution_requests)`, where:
 
 ```python
-def get_execution_requests(execution_requests: list[bytes]) -> ExecutionRequests:
+def get_execution_requests(execution_requests: List[bytes]) -> ExecutionRequests:
```

Comment:
Doesn't Python use lowercase `list`? I believe you were correct with `list[bytes]`

Reply:
seems to be [equivalent in python >= 3.9](https://stackoverflow.com/questions/52629265/static-typing-in-python3-list-vs-list#comment115824266_52629475)

Reply:
Yeah both are correct, but `List` is what the spec uses. So good change & kind of my fault.

Could you change the `list` to `List` in these three spots too?

https://github.com/ethereum/consensus-specs/blob/9752a1cfc065aadca1ff9e8c11267eb9ed42727e/pysetup/spec_builders/electra.py#L30-L33

https://github.com/ethereum/consensus-specs/blob/9752a1cfc065aadca1ff9e8c11267eb9ed42727e/specs/electra/beacon-chain.md?plain=1#L992-L995

https://github.com/ethereum/consensus-specs/blob/9752a1cfc065aadca1ff9e8c11267eb9ed42727e/specs/electra/beacon-chain.md?plain=1#L1148